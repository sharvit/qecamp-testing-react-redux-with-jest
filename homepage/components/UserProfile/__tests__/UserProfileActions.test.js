/* UserProfileActions.test.js */
import { testActionSnapshotWithFixtures } from '../../../../testHelpers';
import {
  updateShowAvatar,
  updateShowPosts,
  updateShowPhotos,
} from '../UserProfileActions';

const fixtures = {
  'should update-show-avatar': () => updateShowAvatar(true),
  'should update-show-posts': () => updateShowPosts(true),
  'should update-show-photos': () => updateShowPhotos(true),
};

describe('UserProfile - Actions', () =>
  testActionSnapshotWithFixtures(fixtures));
