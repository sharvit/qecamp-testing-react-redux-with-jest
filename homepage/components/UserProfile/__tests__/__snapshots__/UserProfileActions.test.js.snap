// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserProfile - Actions should update-show-avatar 1`] = `
Object {
  "payload": true,
  "type": "USER_PROFILE_UPDATE_AVATAR",
}
`;

exports[`UserProfile - Actions should update-show-photos 1`] = `
Object {
  "payload": true,
  "type": "USER_PROFILE_UPDATE_PHOTOS",
}
`;

exports[`UserProfile - Actions should update-show-posts 1`] = `
Object {
  "payload": true,
  "type": "USER_PROFILE_UPDATE_POSTS",
}
`;
